core/src/main/resources/mysql/mysql_init.sql

```diff
@@ -0,0 +1,16 @@
+--
+-- Copyright 2024 Datastrato Pvt Ltd.
+-- This software is licensed under the Apache License version 2.
+--
+
+CREATE TABLE IF NOT EXISTS `metalake_meta`
+(
+    `id` BIGINT(20) UNSIGNED NOT NULL COMMENT 'metalake id',
+    `metalake_name` VARCHAR(128) NOT NULL COMMENT 'metalake name',
+    `metalake_comment` VARCHAR(256) DEFAULT '' COMMENT 'metalake comment',
+    `properties` MEDIUMTEXT DEFAULT NULL COMMENT 'metalake properties',
+    `audit_info` MEDIUMTEXT NOT NULL COMMENT 'metalake audit info',
+    `schema_version` TEXT NOT NULL COMMENT 'metalake schema version info',
+    PRIMARY KEY (`id`),
+    UNIQUE KEY `uk_mn` (`metalake_name`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COMMENT 'metalake metadata';
```

Comment:
The default `COLLATE` is `utf8mb4_0900_ai_ci`? If so, I think we should specifically assign the value as `utf8mb4_0900_ai_ci` or 

```SQL
select * from metalake_meta where metalake_name = 'a'
```

will get metalake `A` and `a`.

Reply:
utf8mb4_0900_ai_cs should only be supported on MySQL 8.0 and above. Should we consider compatibility with MySQL 5.7?

Reply:
>  Should we consider compatibility with MySQL 5.7

I think so. Anyway, I need to consider this scenario and find a solution.

Reply:
Maybe we can set `COLLATE` to `utf8mb4_bin`, which is also supported in MySQL 5.7

Reply:
@qqqttt123 @jerryshao 
What's your opinion on it?

Reply:
Does the MySQL support to judge the version to modify the parameters?

Reply:
@qqqttt123 MySQL should not be able to actively sense the version. If we want to distinguish between multiple versions, we may need to provide a parameter to the user to specify the MySQL version

Reply:
Maybe you can refer to https://blog.csdn.net/lamp_yang_3533/article/details/52266320

Reply:
Yes, we can get the MySQL version number this way. But the pending question is whether we need to provide multiple versions of MySQL table building SQL. If we are sure to provide it, maybe we can let users initialize the corresponding version table DDLs by themselves